copilot/fastapi/run_tests.sh

```
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Colors for terminal output
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+echo -e "${YELLOW}Setting up test environment...${NC}"
+
+# Set working directory to the script location
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)